internal/introspect/introspect.go

```diff
@@ -89,23 +87,14 @@ func GetToken(pkg tokens.Package, i any) (tokens.Type, error) {
 		typ = typ.Elem()
 	}
 
-	var name string
-	var mod string
-	if typ.Kind() == reflect.Func {
-		fn := runtime.FuncForPC(reflect.ValueOf(i).Pointer())
-		parts := strings.Split(fn.Name(), ".")
-		name = parts[len(parts)-1]
-		mod = strings.Join(parts[:len(parts)-1], "/")
```

Comment:
I didn't see this move anywhere else in this review; do we not want this behavior? is is covered somewhere else by the change in layering?

Reply:
We never use this behavior. This was a remnant from when we tried to make Invoke and Construct calls wrappers around functions instead of structs. I forgot to remove it.